sklearn/cluster/k_means_.py

```diff
@@ -175,7 +175,10 @@ def k_means(X, n_clusters, init='k-means++', precompute_distances='auto',
     Parameters
     ----------
     X : array-like or sparse matrix, shape (n_samples, n_features)
-        The observations to cluster.
+        The observations to cluster. It must be noted that, the data
+        will be converted to C ordering if the classical EM-style algorithm
```

Comment:
I would avoid to give details regarding the algorithm:

```
... C ordering when ``algorithm="full"``
```

Reply:
Uhm at the end we always use C-ordered so no need for this specificity.

Reply:
Ah, I forgot to change this back.